app/vmalert/config/config.go

```diff
@@ -41,6 +43,13 @@ type Group struct {
 	XXX map[string]interface{} `yaml:",inline"`
 }
 
+func (g *Group) ParseIntervalOffset() time.Duration {
+	if g.IntervalOffset == nil {
```

Comment:
this check and default value could be moved into 

```go
func (g *Group) UnmarshalYAML()
```
since it already has some default value setting.

Reply:
Agree with @f41gh7 
I also think we need to validate whether IntervalOffset is < than Interval. It doesn't't make sense to have `interval: 60s` and `interval_offset: 70s`